js/build.js

```diff
@@ -263,6 +280,8 @@ Fliplet.Widget.instance('chat', function (data) {
 
     getContacts(false).then(function () {
       return getConversations();
+    }).catch(function(error) {
```

Comment:
@squallstar with that it isn't. I assume only if the `getConversations()` somehow fails, right?